### PR TITLE
NPM test E2E split docker and packages tests

### PR DESCRIPTION
**Pull request by nplanel**
### What does this PR do?

Split docker and packages E2E tests

### CODE REVIEW

File: .gitlab/e2e/e2e.yml [hunk 1]
```diff
@@ -215,17 +215,31 @@ new-e2e-language-detection:
...
-    - qa_agent
     - deploy_deb_testing-a7_x64
     - deploy_rpm_testing-a7_x64
     - deploy_windows_testing-a7
   variables:
     TARGETS: ./tests/npm
     TEAM: network-performance-monitoring
+  parallel:
+    matrix:
+      - EXTRA_PARAMS: --run TestEC2(VM|VMSELinux|VMWKit)Suite
```
Comment (KevinFairise2) [inline .gitlab/e2e/e2e.yml:230]:
```suggestion
    EXTRA_PARAMS: --run TestEC2(VM|VMSELinux|VMWKit)Suite
```

File: .gitlab/e2e/e2e.yml [hunk 2]
```diff
@@ -215,17 +215,31 @@ new-e2e-language-detection:
...
+  extends: .new_e2e_template
+  rules: !reference [.on_npm_or_e2e_changes_or_manual]
+  needs:
+    - qa_agent
+  variables:
+    TARGETS: ./tests/npm
+    TEAM: network-performance-monitoring
+  parallel:
+    matrix:
+      - EXTRA_PARAMS: --run Test(ECSVM|EC2VMContainerized)Suite
```
Comment (KevinFairise2) [inline .gitlab/e2e/e2e.yml:242]:
```suggestion
    EXTRA_PARAMS: --run Test(ECSVM|EC2VMContainerized)Suite
```

**Review by KevinFairise2 — APPROVED** [review]
I think you do not need to create a matrix with only one element. The suggested changes should have the same behavior

**Review by L3n41c — APPROVED** [review]

**Review by pducolin — APPROVED** [review]

**Pull request merged**